Project Mars/MarsQA1/MarsQA-1/Specflow pages/Pages/Language_page.cs

```diff
@@ -0,0 +1,196 @@
+using System;
+using System.Threading;
+using Gherkin.CucumberMessages.Types;
+using MarsQA1.Helper;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using TechTalk.SpecFlow;
+
+namespace MarsQA1.Specflow_Pages;
+
+public class Language_page
+{
+    
+    public void LanguageTab(IWebDriver driver)
+    {
+        Wait.WaitToBeClickable(driver, "XPath","//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div" , 3); 
+        Driver.driver.FindElement(By.XPath(
+            "//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div")).Click();
+    }
+
+    public void AddNew(IWebDriver driver,string language, string dropdownValue)
+    {
+        //Enter the language in the TextBox
+        Driver.driver.FindElement(By.Name("name")).SendKeys(language);
+        
+       //Find the Dropdown Element
+        Wait.WaitToBeClickable(driver, "Name", "level", 5);
+        IWebElement dropdown = Driver.driver.FindElement(By.Name("level"));
+        SelectElement select = new SelectElement(dropdown);
+        select.SelectByText(dropdownValue);
+        
+    }
+
+    public void ClickAddButton()
+    {
+        Driver.driver.FindElement(By.XPath("//*[@id=\'account-profile-section\']/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[3]/input[1]")).Click();
+    }
+
+    public void Add_Verify(IWebDriver driver,string validationMessage)
+    {
+       try
+       {
+           Wait.WaitToBeVisible(driver, "XPath", "//div[@class='ns-box-inner']", 5);
+           IWebElement toastElement = Driver.driver.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+
+           var scenarioTitle = ScenarioContext.Current.ScenarioInfo.Title;
+           switch (scenarioTitle)
+           {
+               case "Scenario1":
```

Comment:
please move asserts to steps, thats where they should be, also how are you testing that your language is added but you have not tested that the langauge you added is present in the table please check this